spec/controllers/idv/hybrid_handoff_controller_spec.rb

```diff
@@ -107,10 +107,18 @@
         skip_upload_step: false }
```

Comment:
suggestion: move the `flow_path` and `destination ` assignments to their respective `it` blocks like you did for `desktop` below, since there are 2 now.